package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "cheerio": "^1.0.0-rc.2",
-    "chokidar": "^2.0.2",
+    "chokidar": "^1.7.0",
```

Comment:
Prolly otta remove the `^`?

Reply:
Eh, I'd say that's unlikely to matter. There aren't any versions of chokidar between 1.7 and 2, and `^` won't pick 2+.